package.json

```diff
@@ -21,19 +21,22 @@
   "scripts": {
     "bench": "pnpm build && node ./bench.cjs",
     "build": "unbuild",
+    "dev": "vitest dev",
     "lint": "eslint --ext .ts .",
     "release": "pnpm test && pnpm build && standard-version && git push --follow-tags && pnpm publish",
-    "test": "pnpm lint"
+    "test": "pnpm lint && vitest run --coverage"
   },
```

Comment:
`dev` command and `test` command are copied from [`package.json` of `unjs/template`](https://github.com/unjs/template/blob/7739a8bf6e71fb4faf1dd7704c7463967ee00eb5/package.json#L22-L29).